tutorial/index.jade

```diff
@@ -1,105 +1,86 @@
-.chapter(data-title='Account Verification Laravel')
-  .step(
-    data-title='Introduction',
-    data-file='src/main/webapp/WEB-INF/web.xml')
+.chapter(data-title='Account Verification Servlets')
+  .step(data-title='Introduction', data-file='src/main/webapp/WEB-INF/web.xml')
     :markdown
       ## Introduction
 
       Ready to implement user account verification in your application? Here's how it works at a high level:
 
-      1. The users begin the registration process by entering their data,
-         including a phone number into a signup form.
-      2. The authentication system sends a one-time password to the user's
-         mobile phone to verify the possession of that phone number.
-      3. The user enters the one-time password into a form before completing
-         registration.
-      4. The user sees a success page and receives an SMS indicating that
-         his/her account has been create.
+      1. The users begin the registration process by entering their data, including a phone number, into a signup form
```

Comment:
Should we use a period at the end of each sentence? Please take a look at http://www.mhhe.com/mayfieldpub/tsw/period.htm